catalog/app/containers/Admin/Packages.js

```diff
@@ -0,0 +1,115 @@
+import {
+  Table,
+  TableBody,
+  TableHeader,
+  TableHeaderColumn,
+  TableRow,
+  TableRowColumn,
+} from 'material-ui/Table';
+import PT from 'prop-types';
+import React, { Fragment } from 'react';
+import { FormattedMessage as FM } from 'react-intl';
+import { Link } from 'react-router';
+import { compose, setPropTypes, setDisplayName } from 'recompose';
+
+import Spinner from 'components/Spinner';
+import api, { apiStatus } from 'constants/api';
+
+import msg from './messages';
+import { branch, formatActivity, formatDate } from './util';
+import ErrorMessage from './ErrorMessage';
+
+
+const packageActivities = [
+  'installs',
+  'previews',
+  // 'deletes',
```

Comment:
here and with `memberActivities` we should delete the commented lines

Reply:
k